service.libraryautoupdate/changelog.txt

```diff
@@ -0,0 +1,246 @@
+Version 1.0.4
```

Comment:
Not needed anymore, kodi only uses `<news>` now. Not blocking, you can keep submitting the old changelog.

Reply:
Good to know. I like to keep it just so I have a running list on my own. Could eventually move this to a GitHub wiki page or something but will keep for now.